app/presenters/summary/sections/overview.rb

```diff
@@ -2,36 +2,50 @@ module Summary
   module Sections
     class Overview < Sections::BaseSection
       def show?
-        !crime_application.in_progress? && super
+        crime_application.case.present? && super
```

Comment:
The Overview section was only shown on completed applications, however this section was the place it made most sense to show the user what they selected for 'Is this application means tested?'

It made sense to me to only show it if there was a case, because by that point the user will have got the answer to this, and a datestamp (which I have also decided there is no reason not to show on an in progress application as it is already shown on the tasklist sidebar)

Reply:
Might be worth wrapping this in a method so that it is clear why it is here. Its not that obvious that the case being pressent means that we should show overview details - unless you are familar with the flow.

```ruby
def show_overview_details?
    crime_application.case.present?
end
```